docs/api.rst

```diff
@@ -25,6 +25,5 @@
 API Docs
 ========
 
-invenio_deposit
----------------
-
+.. automodule:: invenio_deposit.api
```

Comment:
For now just add `.. automodule:: invenio.{views, providers, etc}`. You can include also a titles:

```
Views
-----
```